app/models/claim.rb

```diff
@@ -195,12 +195,12 @@ def approvable?
     submitted? && !payroll_gender_missing? && !decision_made? && !payment_prevented_by_other_claims?
   end
 
-  def decision
-    decisions.last
+  def latest_decision
+    decisions.where(undone: false).last
```

Comment:
we can use the scope here:
```suggestion
    decisions.active.last
```